com.surfshark.Surfshark.yaml

```diff
@@ -0,0 +1,74 @@
+# yaml-language-server: $schema=https://raw.githubusercontent.com/flatpak/flatpak-builder/main/data/flatpak-manifest.schema.json
+
+app-id: com.surfshark.Surfshark
+runtime: org.gnome.Platform
```

Comment:
Does this actually need GNOME runtime or freedesktop will work too?

Reply:
Added GNOME mainly for GObject introspection and GJS